README.md

```diff
@@ -0,0 +1,173 @@
+# Workalendar
```

Comment:
The renaming (and divergent syntax) of the README is going to be a difficult divergence to manage.

This project will need to decide if it wants to adopt the .md format or continue to support the .rst format. I'm much more comfortable with the .rst format, because that's what my other projects use and because it's the default for Python projects. I prefer the md format for it's more universal acceptance. At this point, I don't feel strongly enough about either format to advocate for either, so I'll defer to your preference. If you have no preference, let's go with md, as I think that's the format that will be preferable going forward.

The important thing is going to be maintaining the ancestry and reconciling the changes across the file rename. Here's what I suggest:

- First merge with workalendar at the point right before the rename commit.
- In calendra, rename the file using `git rename`.
- Merge with workalendar after the rename commit and in this merge commit or in a subsequent commit,
- Update any reStructuredText syntax with Markdown syntax.

Reply:
I'm cool with staying on .md, and agree its a right pain to keep in sync. But I'm not clear how to do what you suggest with just this one file (i.e. rewinding the merge would be a LOT of work).

Reply:
I'll take a stab at merging the Calendra-flavoured content by hand into the .md.

Reply:
> But I'm not clear how to do what you suggest with just this one file

Let me take a stab at it. I don't want Git to think these are two different files, else changes in the skeleton to README.rst will show as conflicted with a deleted file instead of conflicted with README.md.

Reply:
I notice that `checkdocs` doesn't work with md-formatted readme.

Reply:
I'm working on a new release of pytest-checkdocs that will at least not fail on markdown content.

Reply:
Fixed in 8f54646168a6e8e665557b93a20661d0ec7eb623.